### PR TITLE
chore(task): improve error message for AccessDenied to secrets

### PR DESCRIPTION
**Pull request by paragbhingre**
This PR improves error messaging when AccessDenied error occurs while accessing the secrets

### CODE REVIEW

File: internal/pkg/cli/task_run.go [hunk 1]
```diff
@@ -578,6 +578,9 @@ func (o *runTaskOpts) Execute() error {
 
 	tasks, err := o.runTask()
 	if err != nil {
+		if strings.Contains(err.Error(), "ResourceInitializationError") && o.appName != "" && o.env != "" {
+			log.Error("It looks like your task is not able to pull the secrets. Did you tag secrets with \"copilot-application\" and \"copilot-environment\" tags \n")
```
Comment (efekarakus) [inline internal/pkg/cli/task_run.go]: nit: Go has a special syntax for "raw string literals" that makes it easier to write these long strings with escaped quotes: https://yourbasic.org/golang/multiline-string/
```suggestion
			log.Error(`It looks like your task is not able to pull the secrets.
Did you tag your secrets with the "copilot-application" and "copilot-environment" tags?
`)
```

File: internal/pkg/cli/task_run.go [hunk 2]
```diff
@@ -578,6 +578,9 @@ func (o *runTaskOpts) Execute() error {
 
 	tasks, err := o.runTask()
 	if err != nil {
+		if strings.Contains(err.Error(), "ResourceInitializationError") && o.appName != "" && o.env != "" {
```
Comment (Lou1415926) [inline internal/pkg/cli/task_run.go]: Do you think we need to be more specific in checking the error, for example check if it contains keywords like "AccessDeniedException" and "unable to retrieve secrets"?

Comment (paragbhingre) [reply internal/pkg/cli/task_run.go]: make sense

**Review by efekarakus — APPROVED** [review]
😍

**Review by huanjani — APPROVED** [review]
:shipit:

**Pull request merged**